### ISSUE TITLE
Uncaught TypeError: Cannot read property 'getDomNode' of undefined

### ISSUE THREAD

**Issue by fyeeme** [facebooknuclide/hyperclick#14]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.19.0-dev-60c1b85ed x64
**Electron**: 1.6.9
**OS**: Mac OS X 10.12.5
**Thrown From**: [hyperclick](https://github.com/facebooknuclide/hyperclick) package 0.0.40

### Stack Trace

Uncaught TypeError: Cannot read property 'getDomNode' of undefined

```
At /Users/Allen/.atom/packages/hyperclick/lib/HyperclickForTextEditor.js:99

TypeError: Cannot read property 'getDomNode' of undefined
    at getLinesDomNode (/packages/hyperclick/lib/HyperclickForTextEditor.js:99:40)
    at removeMouseListeners (/packages/hyperclick/lib/HyperclickForTextEditor.js:105:9)
    at Function.module.exports.Emitter.simpleDispatch (/Users/Allen/Projects/javascript/atom/out/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/Users/Allen/Projects/javascript/atom/out/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:28)
    at HTMLElement.detachedCallback (/Users/Allen/Projects/javascript/atom/out/Atom.app/Contents/Resources/app/src/text-editor-element.js:46:18)
    at HTMLElement.PaneElement.itemRemoved (/Users/Allen/Projects/javascript/atom/out/Atom.app/Contents/Resources/app/src/pane-element.js:187:29)
    at Function.module.exports.Emitter.simpleDispatch (/Users/Allen/Projects/javascript/atom/out/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/Users/Allen/Projects/javascript/atom/out/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:28)
    at Pane.module.exports.Pane.removeItem (/Users/Allen/Projects/javascript/atom/out/Atom.app/Contents/Resources/app/src/pane.js:611:20)
    at Pane.module.exports.Pane.destroyItem (/Users/Allen/Projects/javascript/atom/out/Atom.app/Contents/Resources/app/src/pane.js:683:16)
    at Pane.module.exports.Pane.destroyActiveItem (/Users/Allen/Projects/javascript/atom/out/Atom.app/Contents/Resources/app/src/pane.js:660:12)
    at Workspace.closeActivePaneItemOrEmptyPaneOrWindow (/Users/Allen/Projects/javascript/atom/out/app/src/workspace.js:1430:40)
    at HTMLElement.core:close (/Users/Allen/Projects/javascript/atom/out/Atom.app/Contents/Resources/app/src/register-default-commands.js:231:32)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Users/Allen/Projects/javascript/atom/out/Atom.app/Contents/Resources/app/src/command-registry.js:265:29)
    at /Users/Allen/Projects/javascript/atom/out/Atom.app/Contents/Resources/app/src/command-registry.js:3:59
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Users/Allen/Projects/javascript/atom/out/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:606:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Users/Allen/Projects/javascript/atom/out/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:397:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/Users/Allen/Projects/javascript/atom/out/Atom.app/Contents/Resources/app/src/window-event-handler.js:100:36)
    at HTMLDocument.<anonymous> (/Users/Allen/Projects/javascript/atom/out/Atom.app/Contents/Resources/app/src/window-event-handler.js:3:59)
```

### Commands

```
     -8:39.8.0 editor:newline (input.hidden-input)
  2x -8:37.1.0 core:backspace (input.hidden-input)
     -7:48.8.0 core:move-right (input.hidden-input)
  2x -7:48.3.0 core:select-right (input.hidden-input)
     -7:47.1.0 core:save (input.hidden-input)
     -6:58.9.0 core:backspace (input.hidden-input)
  2x -6:58.4.0 core:save (input.hidden-input)
     -6:40.9.0 core:move-left (input.hidden-input)
     -6:40.6.0 core:move-right (input.hidden-input)
     -6:40.3.0 core:backspace (input.hidden-input)
  4x -6:39.8.0 core:save (input.hidden-input)
 11x -1:39.3.0 core:close (input.hidden-input)
     -1:30 core:paste (input.hidden-input)
     -1:29.5.0 core:save (input.hidden-input)
     -1:27.5.0 core:cut (input.hidden-input)
     -1:27 core:save (input.hidden-input)
```

### Non-Core Packages

```
atom-autocomplete-php 0.22.2 
atom-beautify 0.29.26 
atom-icons 0.2.2 
busy-signal 1.4.1 
custom-app-icon 0.2.3 
docblockr 0.10.3 
git-blame 1.1.1 
highlight-line 0.12.0 
hyperclick 0.0.40 
intentions 1.1.2 
isotope-ui 2.8.5 
last-cursor-position 0.9.2 
linter 2.1.4 
linter-php 1.3.2 
linter-ui-default 1.6.0 
nord-atom-syntax 0.9.1 
nord-atom-ui 0.11.0 
php-cs-fixer 4.1.0 
php-twig 4.0.0 
platformio-ide-terminal 2.5.1 
project-manager 3.3.5 
```

**Comment by fliebe92** [comment]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.19.0-beta0 x64
**Electron**: 1.6.9
**OS**: Mac OS X 10.12.5
**Thrown From**: [hyperclick](https://github.com/facebooknuclide/hyperclick) package 0.0.40


### Stack Trace

Uncaught TypeError: Cannot read property 'getDomNode' of undefined

```
At /Users/flo/.atom/packages/hyperclick/lib/HyperclickForTextEditor.js:99

TypeError: Cannot read property 'getDomNode' of undefined
    at getLinesDomNode (/packages/hyperclick/lib/HyperclickForTextEditor.js:99:40)
    at removeMouseListeners (/packages/hyperclick/lib/HyperclickForTextEditor.js:105:9)
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at HTMLElement.detachedCallback (/Applications/Atom.app/Contents/Resources/app/src/text-editor-element.js:46:18)
    at HTMLElement.PaneElement.itemRemoved (/Applications/Atom.app/Contents/Resources/app/src/pane-element.js:187:29)
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at Pane.module.exports.Pane.removeItem (/Applications/Atom.app/Contents/Resources/app/src/pane.js:634:26)
    at Pane.module.exports.Pane.destroyItem (/Applications/Atom.app/Contents/Resources/app/src/pane.js:706:22)
    at Pane.module.exports.Pane.destroyActiveItem (/Applications/Atom.app/Contents/Resources/app/src/pane.js:683:18)
    at Workspace.closeActivePaneItemOrEmptyPaneOrWindow (/Applications/Atom.app/Contents/Resources/app/src/workspace.js:1703:48)
    at HTMLElement.core:close (/Applications/Atom.app/Contents/Resources/app/src/register-default-commands.js:233:38)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:265:35)
    at /Applications/Atom.app/Contents/Resources/app/src/command-registry.js:3:65
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:610:22)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:401:28)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:100:42)
    at HTMLDocument.<anonymous> (/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:3:65)
```

### Commands

```
     -6:35.1.0 fuzzy-finder:toggle-file-finder (input.hidden-input)
     -6:31.9.0 core:confirm (input.hidden-input)
     -6:15.6.0 project-find:show (input.hidden-input)
  3x -6:11.4.0 core:backspace (input.hidden-input)
  2x -6:08.8.0 find-and-replace:focus-next (input.hidden-input)
  3x -6:07.6.0 core:backspace (input.hidden-input)
     -6:06.4.0 core:confirm (input.hidden-input)
     -5:08.4.0 find-and-replace:show (input.hidden-input)
     -5:07.7.0 core:move-right (input.hidden-input)
     -5:07.5.0 core:confirm (input.hidden-input)
     -4:53 editor:select-to-first-character-of-line (input.hidden-input)
     -4:52.4.0 core:copy (input.hidden-input)
     -0:58.5.0 incompatible-packages:view (div.incompatible-packages-status.inline-block.text.text-error)
     -0:06.6.0 core:close (input.hidden-input)
```

### Non-Core Packages

```
atom-alignment 0.13.0 
atom-bracket-highlight 0.3.2 
atom-clock 0.1.9 
ctags-status 1.3.7 
dqs-shell-commands 2.2.3 
file-icons 2.1.7 
flex-tool-bar 0.12.0 
git-blame 1.1.1 
git-control 0.9.0 
git-history 3.3.0 
git-log 0.4.1 
git-plus 7.9.3 
git-time-machine 1.5.9 
goto-definition 1.3.3 
highlight-line 0.12.0 
highlight-selected 0.13.1 
hyperclick 0.0.40 
keyboard-scroll 0.7.0 
language-generic-config 1.3.0 
language-log 1.8.0 
language-perl-template-toolkit 0.1.0 
less-than-slash 0.16.0 
merge-conflicts 1.4.5 
minimap 4.28.2 
minimap-autohide 0.10.1 
minimap-find-and-replace 4.5.2 
minimap-git-diff 4.3.1 
minimap-highlight-selected 4.6.1 
minimap-split-diff 0.3.7 
open-terminal-here 2.3.0 
pdf-view 0.59.0 
process-palette 0.12.1 
project-manager 3.3.5 
split-diff 1.4.1 
symbols-list 2.4.1 
synced-sidebar 0.4.3 
tab-numbers 0.6.0 
term3 0.22.1 
todo-show 1.11.0 
tool-bar 1.1.0 
tree-view-git-status 1.4.0 
```

**Comment by green3g** [comment]
[Enter steps to reproduce:]

Click on a file in the file tree. 

**Atom**: 1.19.0-beta0 x64
**Electron**: 1.6.9
**OS**: Microsoft Windows 7 Professional 
**Thrown From**: [hyperclick](https://github.com/facebooknuclide/hyperclick) package 0.0.40


### Stack Trace

Uncaught TypeError: Cannot read property 'getDomNode' of undefined

```
At C:\Users\groemhildt\.atom\packages\hyperclick\lib\HyperclickForTextEditor.js:99

TypeError: Cannot read property 'getDomNode' of undefined
    at getLinesDomNode (/packages/hyperclick/lib/HyperclickForTextEditor.js:99:40)
    at addMouseListeners (/packages/hyperclick/lib/HyperclickForTextEditor.js:109:9)
    at Function.module.exports.Emitter.simpleDispatch (~/AppData/Local/atom/app-1.19.0-beta0/resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (~/AppData/Local/atom/app-1.19.0-beta0/resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at HTMLElement.attachedCallback (~/AppData/Local/atom/app-1.19.0-beta0/resources/app/src/text-editor-element.js:42:18)
    at HTMLElement.PaneElement.activeItemChanged (~/AppData/Local/atom/app-1.19.0-beta0/resources/app/src/pane-element.js:144:24)
    at Function.module.exports.Emitter.simpleDispatch (~/AppData/Local/atom/app-1.19.0-beta0/resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (~/AppData/Local/atom/app-1.19.0-beta0/resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at Pane.module.exports.Pane.setActiveItem (~/AppData/Local/atom/app-1.19.0-beta0/resources/app/src/pane.js:337:28)
    at Pane.module.exports.Pane.activateItem (~/AppData/Local/atom/app-1.19.0-beta0/resources/app/src/pane.js:492:27)
    at Workspace.<anonymous> (~/AppData/Local/atom/app-1.19.0-beta0/resources/app/src/workspace.js:1161:20)
    at Generator.next (<anonymous>)
    at step (~/AppData/Local/atom/app-1.19.0-beta0/resources/app/src/workspace.js:11:279)
```

### Commands

```
     -5:57.6.0 tree-view:show (atom-workspace.workspace.scrollbars-visible-always)
     -5:37.3.0 tree-view:add-file (span.name.icon.icon-file-directory)
     -2:58 core:cancel (input.hidden-input)
```

### Non-Core Packages

```
atom-beautify 0.29.26 
atom-jinja2 0.6.0 
atom-typescript 11.0.3 
autocomplete-python 1.8.64 
busy-signal 1.4.3 
docblockr 0.10.3 
file-icons 2.1.7 
git-plus 7.9.3 
git-time-machine 1.5.9 
go-debug 1.4.3 
go-plus 5.5.5 
go-signature-statusbar 1.2.0 
highlight-selected 0.13.1 
hyperclick 0.0.40 
intentions 1.1.2 
kite 0.27.0 
language-batch 0.4.0 
language-markdown 0.23.0 
linter 2.1.4 
linter-eslint 8.2.1 
linter-pycodestyle 2.0.2 
linter-ui-default 1.6.1 
merge-conflicts 1.4.5 
minimap 4.28.2 
minimap-find-and-replace 4.5.2 
minimap-git-diff 4.3.1 
minimap-highlight-selected 4.6.1 
split-diff 1.4.1 
termination 0.5.1 
```

**Comment by OlegSmelov** [comment]
This has been fixed in [nuclide](https://atom.io/packages/nuclide), but not in the standalone package.

Here's a workaround for now: https://github.com/facebooknuclide/hyperclick/issues/4#issuecomment-309225606

**Comment by hansonw** [comment]
Published a new v0.1.0 that fixes this.
